nss-cohort-36/bangazon-workforce-mgt-virtual-magicians
# Employee list 2

# Description
The employee list should now show the correlating department that each employee belongs to.

Closes #19 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. Ensure that connection.py contains the appropriate absolute path for your machine to test locally
2. `git checkout master`
3. `git fetch --all`
4. `git checkout Employee-List-2`
5. If you have yet inserted data for your department and employee table please do so now before testing.
insert the following queries to populate your tables with data to test with:

`INSERT INTO hrapp_department (dept_name, dept_budget)
VALUES ("transport", 35000000);`

`INSERT INTO hrapp_employee (first_name, last_name, start_date, is_supervisor, department_id)
VALUES ('Julian', 'Garcia', '2014-04-05', TRUE, 1);` 

Now you may go to http://localhost:8000/employees/ and see that the employee listed has thie department shown alongside.

# Checklist:
